python/py-pyobjc-quartz/Portfile

```diff
@@ -2,65 +2,23 @@
 
 PortSystem          1.0
 PortGroup           python 1.0
+PortGroup           obsolete 1.0
 
 name                py-pyobjc-quartz
-python.rootname     pyobjc-framework-Quartz
 version             5.3
 revision            0
-categories-append   devel
-platforms           darwin
-license             MIT
-maintainers         nomaintainer
-
-description         Quartz wrappers for PyObjC
-long_description    This port provides the CoreGraphics, ImageIO, \
-                    QuartzComposer, QuartzCore, QuartzFilters, ImageKit, \
-                    PDFKit and CoreVideo framework wrappers \
-                    for use with PyObjC.
-homepage            https://pyobjc.readthedocs.io/
-
-master_sites        pypi:p/${python.rootname}/
-distname            ${python.rootname}-${version}
-
-checksums           rmd160  a06bc0f1da8559473be5b420a462693483487351 \
-                    sha256  fb89be73ca9dff503375e7a73be5a957bf97c8308259ab733ddfdcc2f70a3fa4 \
-                    size    3372445
-
-python.versions     27 34 35 36 37 38
-
-if {${name} ne ${subport}} {
-    depends_build-append \
-                        port:py${python.version}-setuptools
-    depends_lib-append  port:py${python.version}-pyobjc \
-                        port:py${python.version}-pyobjc-cocoa
+epoch               1
 
-    patchfiles      pyobjc_setup.py.patch
+categories          python devel
+license             MIT
 
-    if {${configure.sdkroot} eq ""} {
-        set sdkroot "/"
-    } else {
-        set sdkroot ${configure.sdkroot}
-    }
+replaced_by         py-pyobjc
 
-    build.env-append    SDKROOT=${sdkroot}
-    destroot.env-append SDKROOT=${sdkroot}
+description         obsolete wrapper for PyObjC
 
-    platform darwin 8 {
-        depends_build
-        depends_lib
-        pre-fetch {
-            ui_error "${subport} @${version} requires Mac OS X 10.5 or later"
-            return -code error "incompatible Mac OS X version"
-        }
+foreach v {27 34 35 36 37 38} {
```

Comment:
please remove the py34 /py35subports; you cannot set it to be replaced_by py3[45]-pyobjc as that port doesn't define the py3[45] subports